package.json

```diff
@@ -112,27 +112,32 @@
         "compile": "tsc -p ./ && node copyStaticFiles.js",
         "watch": "node copyStaticFiles.js && tsc -watch -p ./",
         "postinstall": "node ./node_modules/vscode/bin/install",
-        "test": "npm run compile && node ./node_modules/vscode/bin/test",
```

Comment:
This earlier command was pretty interesting as in they go run test binaries from under /node_modules/vscode. What are those test supposed to be?